rest-api-spec/src/main/resources/rest-api-spec/api/indices.resolve_index.json

```diff
@@ -58,7 +58,6 @@
           "logsdb",
           "lookup"
         ],
-        "default": "",
```

Comment:
This one is unrelated to the above. Due to a [limitation of the Elasticsearch specification tooling](https://github.com/elastic/elasticsearch-specification/blob/cbedec96dc721386685762da6d5690c8aa25b501/compiler/src/model/utils.ts#L851), we can't currently set a default on this one.